fast64_internal/sm64/__init__.py

```diff
@@ -228,13 +228,17 @@ def upgrade_changed_props():
             bpy.context.scene.fast64.sm64.exportType = get_legacy_export_type()
             bpy.context.scene.fast64.sm64.version = SM64_Properties.cur_version
         # props upgrade for combined export panel
-        if hasattr(bpy.context.scene, "geoName"):
+        if bpy.context.scene.get("geoName", None):
```

Comment:
If you are already have defaults for all the props this is very much unimportant and again, I'm pretty sure that unregistered default values are always going to be None so honestly just get rid of this check I guess, and another thing, you aren´t changing the version so this will run after all file loads, lets not do that.